node/directives/withCurrentProfile.ts

```diff
@@ -22,6 +22,45 @@ export class WithCurrentProfile extends SchemaDirectiveVisitor {
         return null
       }
 
+      const {
+        vtex: { storeUserAuthToken, adminUserAuthToken, account },
+        dataSources: { identity },
+      } = context
+
+      const tokenUser = await identity.getUserWithToken(
+        storeUserAuthToken ?? adminUserAuthToken ?? ''
+      )
+
+      if (tokenUser.account !== account) {
+        const {
+          fieldName,
+          returnType,
+          operation: { operation, name },
+        } = info
+
+        const {
+          vtex: { logger, host },
+          req: {
+            headers: { 'user-agent': userAgent, referer },
+          },
+        } = context
+
+        const logData = {
+          host,
+          referer,
+          userAgent,
+          message: 'Type: CrossTokenAccount',
+          account,
+          tokenAccount: tokenUser.account ?? '',
+          caller: tokenUser.user ?? '',
+          fieldName,
+          fieldType: (returnType as any).name,
+          operation: `${operation} ${name?.value ?? ''}`,
```

Comment:
![Screen Shot 2021-11-03 at 09 41 05](https://user-images.githubusercontent.com/10627086/140061725-7e264fc0-15de-46e5-bf0d-d9dd312aafe3.png)

to avoid having white space.

another alternative is to use `trim()`

```suggestion
          operation: name?.value ? `${operation} ${name?.value}` : operation,
```